diagnostic.js

```diff
@@ -41,7 +79,9 @@ db.dropDatabase();
 // House Arryn is not honorable!  Delete their motto.
 
 // your code begins here
-
+db.houses.remove(
```

Comment:
this doesn't just delete their motto, this deletes everything associated with whatever fit's the motto